packages/legend-application-repl/src/stores/dataCube/DataCubeState.ts

```diff
@@ -44,9 +45,8 @@ class DataCubeTask {
 }
 
 export class DataCubeState {
-  readonly repl: REPLStore;
-  readonly application: LegendREPLApplicationStore;
-  readonly engine: DataCubeEngine;
+  readonly dataCubeStore: DataCubeStore;
```

Comment:
Trying to keep thing less verbose in data cube codebase

```suggestion
  readonly store: DataCubeStore;
```